certbot/tests/compat/misc_test.py

```diff
@@ -0,0 +1,44 @@
+"""Tests for certbot.compat.misc"""
+import mock
```

Comment:
This  needs to be updated now to fallback to `unittest.mock` like in https://github.com/certbot/certbot/pull/7911.

Reply:
Done.